src/inspector/node_protocol.pdl

```diff
@@ -71,6 +71,11 @@ experimental domain NodeWorker
   # Detaches from all running workers and disables attaching to new workers as they are started.
   command disable
 
+  # Detached from worker with given sessionId.
```

Comment:
I added `the` since it sounds like there is `the` worker with given sessionId but I am open for any input from more proficient in English colleagues :)